### ISSUE TITLE
/dev/snd is not showing hot plugged USB device

### ISSUE THREAD

**Issue by rajasec** [moby/moby#39262]
**Description**

Briefly describe the problem you are having in a few paragraphs.
we have one of our services running in docker container where we need to play the sound using alsa. We have mounted /dev/snd before starting the container, we are able to play successfully
if we have inserted USB based new mic and speaker /dev/snd is updated in Host, but it is not reflected in docker container. is there any solution for dynamic plug of devices which can reflect in host 

**Steps to reproduce the issue:**
1. docker run -ti -v /dev/snd:snd hwservice
2. insert the new USB based MIC and speaker in console
3. /dev/snd directory is not having the new device node 

**Describe the results you received:**
/dev/snd is not having the new sound card

**Describe the results you expected:**
/dev/snd shall contain the new card information which has been hot plugged 

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

**Output of `docker info`:**
Server Version: 17.03.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: false
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 4ab9917febca54791c5f071a9d1f404867857fcc
runc version: 54296cf40ad8143b62dbcaa1d90e520a2136ddfe
init version: 949e6fa
Kernel Version: 4.1.12-124.21.1.el6uek.x86_64
Operating System: <unknown>
OSType: linux
Architecture: x86_64
CPUs: 12
Total Memory: 31.06 GiB
Name: magic
ID: FXLO:NDQ7:HZLL:XVE6:A4U7:QWR6:G2AJ:KI7E:R5RS:XHID:S3XB:DEZL
Docker Root Dir: /xrayDigital/docker-data
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: IPv4 forwarding is disabled
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by thaJeztah** [comment]
Looks like a duplicate of https://github.com/moby/moby/issues/27807 / https://github.com/moby/moby/issues/34290, which has some workarounds / useful comments

Let me close this one to not have the conversation diverge, but feel free to continue the conversation
